app/views/tag/_advanced_tagging.html.erb

```diff
@@ -3,31 +3,31 @@
 </button>
 <div class="dropdown-menu dropdown-menu-right float-right" style="margin-top:5px;" id="pt-list">
   <% if current_user %>
-    <p class="dropdown-header"><b>Recently used tags</b></p>
+    <p class="dropdown-header"><b><%= translation('tag.advanced_tagging.recently_used_tags') %></b></p>
     <% NodeTag.includes(:tag).where(uid: current_user.id).order('date DESC').limit(5).each do |tag| %>
       <a class="dropdown-item" onClick="addTag('<%= tag.name %>')"><span style="cursor:pointer;" class="badge badge-primary"><%= tag.name %></span></a>
     <% end %>
   <% end %>
   <div class="dropdown-divider"></div>
   <% if @node && @node.type == "page" %>
-    <a class="dropdown-item" href="#" data-value="parent:">Link to a "parent" page (wikis only)</a>
-    <a class="dropdown-item" href="#" data-value="style:">Add style</a>
-    <a class="dropdown-item" href="#" data-value="lang:">Add Translation widget</a>
++    <a class="dropdown-item" href="#" data-value="parent:"><%= translation('tag.advanced_tagging.link_to_parent') %></a>
++    <a class="dropdown-item" href="#" data-value="style:"><%= translation('tag.advanced_tagging.add_style') %></a>
++    <a class="dropdown-item" href="#" data-value="lang:"><%= translation('tag.advanced_tagging.add_translation_widget') %></a>
   <% else %>
-    <a class="dropdown-item" href="#" data-value="seeks:photos">Request response photos</a>
-    <a class="dropdown-item" href="#" data-value="hidden:response">Hide responses from dashboard</a>
-    <a class="dropdown-item" href="#" data-value="with:">Add Coauthor (notes only)</a>
-    <a class="dropdown-item" href="#" data-value="series:">Mark as part of a series</a>
-    <a class="dropdown-item" href="#" data-value="comment-template:">Add template to the comment form</a>
-    <a class="dropdown-item" target="_blank" href="/wiki/comment-templates">Write a template for the comment form</a>
++    <a class="dropdown-item" href="#" data-value="seeks:photos"><%= translation('tag.advanced_tagging.request_photos') %></a>
++    <a class="dropdown-item" href="#" data-value="hidden:response"><%= translation('tag.advanced_tagging.hide_responses') %></a>
++    <a class="dropdown-item" href="#" data-value="with:"><%= translation('tag.advanced_tagging.add_coauthor') %></a>
++    <a class="dropdown-item" href="#" data-value="series:"><%= translation('tag.advanced_tagging.mark_as_series') %></a>
++    <a class="dropdown-item" href="#" data-value="comment-template:"><%= translation('tag.advanced_tagging.add_template') %></a>
++    <a class="dropdown-item" target="_blank" href="/wiki/comment-templates"><%= translation('tag.advanced_tagging.write_template') %></a>
```

Comment:
Hi @SehgalNeeraj, can you remove the `+` signs in front of each added line? Thanks!

Reply:
Yeah @SehgalNeeraj remove the + signs and it is ready to merge then

Reply:
@TildaDares  @govindgoel  I have made changes and committed to the same PR.
Can you review and suggest next steps.

Reply:
Still it's the same

![Screenshot from 2022-01-20 17-37-48](https://user-images.githubusercontent.com/52847415/150335970-222c0ac2-e0ef-40e5-8933-642defb0233b.png)

Reply:
> Still it's the same
> 
> ![Screenshot from 2022-01-20 17-37-48](https://user-images.githubusercontent.com/52847415/150335970-222c0ac2-e0ef-40e5-8933-642defb0233b.png)

hi @govindgoel @TildaDares Just want to be sure if you are reviewing the latest [commit](https://github.com/publiclab/plots2/pull/10659/commits/21158b03bafb04b7ada686d8a9361f6adf8e9f0f). I believe its a presentation thing where new added lines are "shown" with single + sign in github.
The code review comments were to for double + (++) sign and I could see the extra + sign in my repo as well. 
Now its removed based on code review comments.

Worst case, I can just abandon this PR and re-raise a fresh one. Thoughts ?

Reply:
Now it's fine